superset-frontend/src/dashboard/reducers/getInitialState.js

```diff
@@ -108,118 +108,116 @@ export default function (bootstrapData) {
   const sliceIds = new Set();
   dashboard.slices.forEach(slice => {
     const key = slice.slice_id;
-    if (['separator', 'markup'].indexOf(slice.form_data.viz_type) === -1) {
-      const form_data = {
-        ...slice.form_data,
-        url_params: {
-          ...slice.form_data.url_params,
-          ...urlParams,
-        },
-      };
-      chartQueries[key] = {
-        ...chart,
-        id: key,
-        form_data,
-        formData: applyDefaultFormData(form_data),
-      };
-
-      slices[key] = {
-        slice_id: key,
-        slice_url: slice.slice_url,
-        slice_name: slice.slice_name,
-        form_data: slice.form_data,
-        edit_url: slice.edit_url,
-        viz_type: slice.form_data.viz_type,
-        datasource: slice.form_data.datasource,
-        description: slice.description,
-        description_markeddown: slice.description_markeddown,
-        owners: slice.owners,
-        modified: slice.modified,
-        changed_on: new Date(slice.changed_on).getTime(),
-      };
+    const form_data = {
```

Comment:
correct. the only change is just removed if viz_type change. I didn't touch any other code.